packages/components/src/promo-card/index.tsx

```diff
@@ -12,13 +12,17 @@ export default function PromoCard( { className, isMobile }: PromoCardProps ) {
 	// Using useTranslate() with interpolation to set up the title/message.
 	// https://wpcalypso.wordpress.com/devdocs/packages/i18n-calypso/README.md
 	const translate = useTranslate();
+	const redirectLink = 'https://jetpack.com/app/';
 	const components = {
-		a: <a href="https://wp.com/apps" />,
+		a: <a href={ redirectLink } />,
 	};
 	// TODO: Fix mobile logic.
 	// Should this maybe come via a media query in the CSS?
```

Comment:
We could definitely use media queries to show the appropriate copy & image combination.

Reply:
I'll add a note about this one to the [main issue](https://github.com/Automattic/wp-calypso/issues/69230) so I remember to circle back to this.